packages/client/hmi-client/src/stores/auth.ts

```diff
@@ -1,115 +1,66 @@
 import { defineStore } from 'pinia';
-import { config } from 'vue-gtag';
+import { User } from '@/types/Types';
+import Keycloak from 'keycloak-js';
+import axios, { AxiosHeaders } from 'axios';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
Please use [randomUUID](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)
```suggestion
import { randomUUID } from crypto
```

Reply:
Note there are some quirky caveats with randomUUID 
- If not explicit import and this is used in a unit-test it might fail because node may not have access to crypto
- If you do explicit as you had above you need to install the package I believe

Upgrading to node version-20 might solve this issue, as I couldn't reproduce it locally (node-20) but Tom could (node-18).

Reply:
why should we use randomUUID? we do not use it anywhere else. I suggest that we do not use randomUUID

Reply:
`randomUUID` is not a library is a WebAPI, so we can remove `uuid` which is a JS dependency.
See the link in my first comment.

Reply:
I will just remove it myself on its own PR trough the whole project, not important at all.

Reply:
we do not use RandomUUID anywhere else. Will stick with uuidv4